subcommand/injector/flags.go

```diff
@@ -132,6 +135,8 @@ func (c *Command) init() {
 		"Path under which to expose metrics")
 	c.flagSet.BoolVar(&c.flagUseLeaderElector, "use-leader-elector", agent.DefaultAgentUseLeaderElector,
 		fmt.Sprintf("Use leader elector to coordinate multiple replicas when updating CA and Certs with auto-tls"))
+	c.flagSet.StringVar(&c.flagDefaultTemplate, "default-template", "map",
+		"Sets the default template type (map or json). Defaults to map.")
```

Comment:
Should the default be referencing a variable like the other flags do? i.e.

```suggestion
	c.flagSet.StringVar(&c.flagDefaultTemplate, "default-template", agent.DefaultTemplate,
		fmt.Sprintf("Sets the default template type (map or json). Defaults to %s.", agent.DefaultTemplate)
```

Reply:
Fixed.